ClosedXML/Excel/Cells/XLCell.cs

```diff
@@ -441,16 +441,22 @@ private object RecalculateFormula(string fA1)
                 cAddress = fA1;
             }
 
-            if (Worksheet.Workbook.WorksheetsInternal.Any<XLWorksheet>(
-                w => String.Compare(w.Name, sName, true) == 0)
-                && XLHelper.IsValidA1Address(cAddress)
-                )
-            {
-                var referenceCell = Worksheet.Workbook.Worksheet(sName).Cell(cAddress);
-                if (referenceCell.IsEmpty(XLCellsUsedOptions.AllContents))
-                    return 0;
-                else
-                    return referenceCell.Value;
+            if (Worksheet.Workbook.Worksheets.Contains(sName)
+                && XLHelper.IsValidA1Address(cAddress))
+            {
+                try
```

Comment:
I know that `try/catch` blocks severely affect performance. But does `try/finally` without `catch` perform well?

Reply:
From what I know, the most performance hit is caused by unwinding stack trace so without the exception there should be no difference. But you've raised an interesting question, i'll try to do a little of benchmarking.

Reply:
https://stackoverflow.com/a/4107320